examples/experimental/onnx/README.md

```diff
@@ -0,0 +1,34 @@
+
+# Classification sample
```

Comment:
Can you revise this and describe the steps  required to reproduce the results in the table (empty now):
- Install
- Run quantization
- Measure accuracy of the baseline and quantized models
- Measure performance of the baseline and quantized models

The table below should contain only accuracy claims (legal restrictions).

Reply:
Readme was updated. Please, take a look

Reply:
Now the readme is ready for review